x-pack/plugins/canvas/public/components/workpad_shortcuts/workpad_shortcuts.tsx

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { Component, KeyboardEvent } from 'react';
+
+import isEqual from 'react-fast-compare';
+// @ts-ignore
+import { Shortcuts } from 'react-shortcuts';
+// @ts-ignore
+import { isTextInput } from '../../lib/is_text_input';
+
+export interface Props {
+  cutNodes: () => void;
+  copyNodes: () => void;
+  pasteNodes: () => void;
+  cloneNodes: () => void;
+  deleteNodes: () => void;
+  bringToFront: () => void;
+  bringForward: () => void;
+  sendBackward: () => void;
+  sendToBack: () => void;
+  groupNodes: () => void;
+  ungroupNodes: () => void;
+  createCustomElement: () => void;
```

Comment:
All without args and without returns?  Remember: these serve as documentation as much as they serve to silence TS errors.

Reply:
Same deal here as in `sidebar_header.tsx`. I've added comments for each handler.